src/strategies/offer_maker/market_making/kandel/AaveKandel.sol

```diff
@@ -91,11 +104,10 @@ contract AaveKandel is GeometricKandel {
     super.withdrawFunds(baseAmount, quoteAmount, recipient);
   }
 
-  ///@notice returns available funds for a particular offer type
   ///@inheritdoc AbstractKandel
   function reserveBalance(OfferType ba) public view override returns (uint balance) {
     IERC20 token = outboundOfOfferType(ba);
-    return pooledRouter().balanceOfReserve(token, RESERVE_ID) + token.balanceOf(address(this));
+    balance = super.reserveBalance(ba) + pooledRouter().balanceOfReserve(token, RESERVE_ID);
```

Comment:
For shared reserves it becomes a bit difficult to calculate the total reserve across all contracts, we would have to take the reserveBalance of individual contracts without what is on the router.